### PR TITLE
Translate fake skill name in schizophrenia

### PR DESCRIPTION
**Pull request by Night-Pryanik**
#### Summary
```SUMMARY: None```

#### Describe the solution
Replaced calling skill name with calling translated skill name.

### CODE REVIEW

File: src/player.cpp [hunk 1]
```diff
@@ -5354,7 +5354,7 @@ void player::suffer()
                 // Skill raise
                 if( !done_effect && one_in( to_turns<int>( 12_hours ) ) ) {
                     skill_id raised_skill = Skill::random_skill();
-                    add_msg( m_good, _( "You increase %1$s to level %2$d" ), raised_skill.c_str(),
+                    add_msg( m_good, _( "You increase %1$s to level %2$d" ), raised_skill.obj().name(),
```
Comment (Ilysen) [inline src/player.cpp]: You can tell it's fake due to the lack of a period at the end of the message; is that intentional?

Comment (Night-Pryanik) [reply src/player.cpp]: I guess no. Will fix.

Comment (Ilysen) [reply src/player.cpp]: I-It was actually an exclamation mark, I'm just dumb. This implementation will still probably fool folks, though.

Comment (Night-Pryanik) [reply src/player.cpp]: > I-It was actually an exclamation mark

No it's not. "Real" message ends with a period.
https://github.com/CleverRaven/Cataclysm-DDA/blob/25e20e22260af33f07dbd853f19e64febf1c77d7/src/player.cpp#L9865

Comment (Ilysen) [reply src/player.cpp]: Oh, my bad.

**Pull request merged**